cmd/monaco/deploy/internal/logging/logging.go

```diff
@@ -14,7 +14,7 @@
  * limitations under the License.
  */
 
-package deploy
+package logging
```

Comment:
note: didn't want to call this `log` to not overlap with the actual `log` package. Would have made for nicer function names though...